addons/udes_stock/models/res_users.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from odoo import api, fields, models, SUPERUSER_ID, _
+from odoo.exceptions import ValidationError
+
+
+class ResUser(models.Model):
+    _inherit = "res.users"
+
+    @api.model
+    def get_user_warehouse(self, aux_domain=None):
```

Comment:
Maybe the name of method as we may expect more than one warehouse and inside when we search it is warehouse = self.search .. should be plural :)
```suggestion
    def get_user_warehouses(self, aux_domain=None):
```

Reply:
I don't believe there are any plans for multi-warehouse systems in the future

Reply:
On the code there is a control if len > 1 and aux_domain 
To my understanding it means that we may expect some warehouses, its not related to multi company.
Event the docstring says warehouse-s smth like that